src/components/AssetsFilters/index.jsx

```diff
@@ -32,22 +30,18 @@ AssetsFilters.propTypes = {
   assetsFilters: PropTypes.shape({
     assetTypes: PropTypes.object,
   }).isRequired,
+  courseDetails: PropTypes.shape({
```

Comment:
This is just pulling the mapStateToProps, mapDispatchToProps, and connect out of AssetsFilters/index.jsx into AssetsFilters/container.jsx; there's no new code